core/trino-spi/src/test/java/io/trino/spi/predicate/TestSortedRangeSet.java

```diff
@@ -157,6 +166,9 @@ public void testUnboundedSet()
         assertTrue(rangeSet.containsValue(0L));
         assertTrue(rangeSet.containsValue(4L));
         assertFalse(rangeSet.containsValue(7L));
+        assertEquals(
+                rangeSet.toString(),
+                "SortedRangeSet[type=bigint, ranges=3, {(min, 0], (1, 6), (9, max)}]");
```

Comment:
This and others, doesn't seem to belong to `Add limit to Domain#toString SPI`.
Perhaps add the tests as the first commit, so that subsequent changes show how the toString is changing within them.